roles/setup_base_config/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+
+- name: stat {{ deployed_tool_versions }}
+  stat:
+    path: "{{ deployed_tool_versions }}"
+  register: stat_result
+
+- name: Create deployed tools version file
+  copy:
+    dest: "{{ deployed_tool_versions }}"
+    content: "-- {{ deployment_environment }} ({{ deployment_version }}) --"
+  when: not stat_result.stat.exists
+
+- name: Create ngi_pipeline conf directory
+  file:
+    path: "{{ ngi_pipeline_conf }}"
+    state: directory
+
+- name: Deploy common bash environment variables
+  template:
+    src: "{{ bash_env_common_script }}.j2"
+    dest: "{{ ngi_pipeline_conf }}/{{ bash_env_common_script }}"
```

Comment:
should this have `force: no` as a parmeter? (also the statement the below)

Reply:
The current monthly branch has the "force: no" removed. So I removed them here as well. (there is a separate commit for this). I actually had some hard work finding these differences :) . Should I put force: no back?

Reply:
Oh, wait, it was I that added the force: no parameter. Thank you for finding it.